CHANGES

```diff
@@ -3,6 +3,37 @@ Version 5.27.0
 
 * Updated CA bundle.
 
+* Raven transports that have dependencies outside Python stdlib are no longer
+  imported by default, and must now be explicitly imported by the application.
+  The transports no longer imported by default are:
+    - raven.transport.eventlet.EventletHTTPTransport
+    - raven.transport.gevent.GeventedHTTPTransport
+    - raven.transport.requests.RequestsHTTPTransport
+    - raven.transport.threaded_requests.ThreadedRequestsHTTPTransport
+    - raven.transport.twisted.TwistedHTTPTransport
+    - raven.transport.tornado.TornadoHTTPTransport
+
+  If you really need to have those transports available, you need to explicitly
+  import them and register them.  For full compatibility:
+  ```
+  from raven.transport.eventlet import EventletHTTPTransport
+  from raven.transport.gevent import GeventedHTTPTransport
+  from raven.transport.requests import RequestsHTTPTransport
+  from raven.transport.twisted import TwistedHTTPTransport
+  from raven.transport.tornado import TornadoHTTPTransport
+  from raven.transport.threaded_requests import ThreadedRequestsHTTPTransport
+  from raven.base import Client
+
+  for transport in [EventletHTTPTransport,
```

Comment:
But the problem is that a typical Django setup just adds `raven.contrib.django.raven_compat` to the list of INSTALLED_APPS.  If they also have a weird DSN configured, they would need to register the transports in the registry for it to work, as the Django Sentry app creates its own Client() and I honestly have no idea how to customize creation of this Client()...